config.yaml

```diff
@@ -26,7 +26,7 @@ options:
   saml_target_url:
     type: string
     description: "SAML authentication target url."
-    default: ""
+    default: "https://login.staging.ubuntu.com/+saml"
```

Comment:
This shouldn't be the default, let's leave the default as "".

Reply:
Fixed, thanks!